client/containers/Pub/usePubBodyState.ts

```diff
@@ -40,15 +40,15 @@ export const usePubBodyState = (options: Options): PubBodyState => {
 	} = usePageContext();
 
 	const discussionAnchors = useMemo(() => {
-		if (isRelease) {
+		if (isRelease || isAVisitingCommenter) {
```

Comment:
i am not sure why i need to do this for new route because draft pubs just render discussion groups